k8s/job.jsonnet

```
@@ -161,5 +162,93 @@ local bigquery_arg = [
     '0.1656',
     '--price-gbp-per-kwh-oil',
     '0.0702',
-  ] + bigquery_arg),
+  ]),
+
+  // sensitivity tests
+
+  job('50-%s-pump-awareness-low' % std.extVar('SHORT_SHA'), [
+    '--heat-pump-awareness',
+    '0.2',
+  ]),
+  job('51-%s-pump-awareness-high' % std.extVar('SHORT_SHA'), [
+    '--heat-pump-awareness',
+    '1.0',
+  ]),
+
+  job('52-%s-annual-reno-rate-low' % std.extVar('SHORT_SHA'), [
+    '--annual-renovation-rate',
+    '0.05',
+  ]),
+  job('53-%s-annual-reno-rate-high' % std.extVar('SHORT_SHA'), [
+    '--annual-renovation-rate',
+    '0.33',
+  ]),
+
+  job('54-%s-household-lookahead-low' % std.extVar('SHORT_SHA'), [
+    '--household-num-lookahead-years',
+    '1',
+  ]),
+  job('55-%s-household-lookahead-high' % std.extVar('SHORT_SHA'), [
+    '--household-num-lookahead-years',
+    '5',
+  ]),
+
+  job('56-%s-heating-system-hassle-low' % std.extVar('SHORT_SHA'), [
+    '--heating-system-hassle-factor',
+    '0',
+  ]),
+  job('57-%s-heating-system-hassle-high' % std.extVar('SHORT_SHA'), [
+    '--heating-system-hassle-factor',
+    '0.2',
+  ]),
+
+  job('58-%s-heat-pump-suitable-low' % std.extVar('SHORT_SHA'), []),
+  job('59-%s-heat-pump-suitable-high' % std.extVar('SHORT_SHA'), [
+    '--all-agents-heat-pump-suitable',
+  ]),
+
+  job('59-%s-gas-price-low' % std.extVar('SHORT_SHA'), [
+    '--price-gbp-per-kwh-gas',
+    '0.02325',
+  ]),
+  job('60-%s-gas-price-high' % std.extVar('SHORT_SHA'), [
+    '--price-gbp-per-kwh-gas',
+    '0.06975',
+  ]),
+
+  job('61-%s-electricity-price-low' % std.extVar('SHORT_SHA'), [
+    '--price-gbp-per-kwh-electricity',
+    '0.1003',
+  ]),
+  job('62-%s-electricity-price-high' % std.extVar('SHORT_SHA'), [
+    '--price-gbp-per-kwh-electricity',
+    '0.3009',
+  ]),
+
+  job('63-%s-oil-price-low' % std.extVar('SHORT_SHA'), [
+    '--price-gbp-per-kwh-oil',
+    '0.0241',
+  ]),
+  job('64-%s-oil-price-high' % std.extVar('SHORT_SHA'), [
+    '--price-gbp-per-kwh-oil',
+    '0.0723',
+  ]),
```

Could we change this to only sensitivity test the tuple of values, instead:
```suggestion
  job('59-%s-fuel-price-low' % std.extVar('SHORT_SHA'), [
    '--price-gbp-per-kwh-gas',
    '0.02325',
    '--price-gbp-per-kwh-electricity',
    '0.1003',
    '--price-gbp-per-kwh-oil',
    '0.0241',
  ]),
  job('60-%s-fuel-price-high' % std.extVar('SHORT_SHA'), 
    '--price-gbp-per-kwh-gas',
    '0.06975',
    '--price-gbp-per-kwh-electricity',
    '0.3009',
    '--price-gbp-per-kwh-oil',
    '0.0723',
  ]),
```

---

If we do this we could renumber all of the scenarios below, but it doesnt matter that much